File: SSM/SSM_PET/src/cs/SSM_PET/petState.cs
using System;
using System.Collections.Generic;
public class PetState 
{
    private double _pet;
    
    /// <summary>
    /// Constructor of the petState component")
    /// </summary>  
    public petState() { }
    
    
    public petState(petState toCopy, bool copyAll) // copy constructor 
    {
        if (copyAll)
        {
    
            pet = toCopy.pet;
        }
    }
    public double pet
    {
        get { return this._pet; }
        set { this._pet= value; } 
    }
}

File: SSM/SSM_PET/src/java/SSM_PET/petState.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;
public class petState
{
    private double pet;
    
    public petState() { }
    
    public petState(petState toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.pet = toCopy.getpet();
        }
    }
    public double getpet()
    { return pet; }

    public void setpet(double _pet)
    { this.pet= _pet; } 
    
}